CppExport/src/visitors/source_visitors/DeclarationVisitorSource.cpp

```diff
@@ -121,54 +121,9 @@ SourceFragment* DeclarationVisitorSource::visitTopLevelClass(Class* classs)
 SourceFragment* DeclarationVisitorSource::visit(Class* classs)
 {
 	auto fragment = new CompositeFragment(classs);
-	if (Class::ConstructKind::Class == classs->constructKind())
-		*fragment << printAnnotationsAndModifiers(classs) << "class " << classs->nameNode();
-	else if (Class::ConstructKind::Interface == classs->constructKind())
-		*fragment << printAnnotationsAndModifiers(classs) << "interface " << classs->nameNode();
-	else if (Class::ConstructKind::Enum == classs->constructKind())
-		*fragment << printAnnotationsAndModifiers(classs) << "enum " << classs->nameNode();
-	else if (Class::ConstructKind::Annotation == classs->constructKind())
-		*fragment << printAnnotationsAndModifiers(classs) << "@interface " << classs->nameNode();
-	else
-		notAllowed(classs);
-
-	if (!classs->typeArguments()->isEmpty())
-		*fragment << list(classs->typeArguments(), ElementVisitorSource(data()), "typeArgsList");
-
-	if (!classs->baseClasses()->isEmpty())
-	{
-		if (Class::ConstructKind::Interface == classs->constructKind() ||
-			 Class::ConstructKind::Annotation == classs->constructKind())
-		{
-			*fragment << " extends ";
-			*fragment << list(classs->baseClasses(), ExpressionVisitorSource(data()), "comma");
-		}
-		else if (Class::ConstructKind::Enum == classs->constructKind())
-		{
-			*fragment << " implements ";
-			*fragment << list(classs->baseClasses(), ExpressionVisitorSource(data()), "comma");
-		}
-		else
-		{
-			// TODO: there might not be an extend and only implements.
-			*fragment << " extends ";
-			*fragment << expression(classs->baseClasses()->at(0));
-
-			if (classs->baseClasses()->size() > 1)
-			{
-				*fragment << " implements ";
-				int i = 1;
-				for (; i < classs->baseClasses()->size() - 1; ++i)
-					*fragment << expression(classs->baseClasses()->at(i)) << ", ";
-				*fragment << expression(classs->baseClasses()->at(i));
-			}
-		}
-	}
-
-	notAllowed(classs->friends());
 
 	//TODO
-	auto sections = fragment->append( new CompositeFragment(classs, "bodySections"));
+	auto sections = fragment->append( new CompositeFragment(classs, "sections"));
```

Comment:
Where did all the code before this line go? We'll still to handle some of those cases (e.g. enums, even though they work very differently compared to java).